_episodes/06-working-with-columns.md

```diff
@@ -26,8 +26,6 @@ You can rename a column by opening the drop-down menu at the top of the column t
 ## Sorting data
 You can sort data in OpenRefine by clicking on the drop-down menu for the column you want to sort on, and choosing `Sort`.
 
-Once you have sorted the data, a new `Sort` drop-down menu will be displayed.
```

Comment:
When a sort has been applied in OpenRefine a completely new "Sort" drop-down menu is added to the UI at the top of the grid. It's important that we keep this information in the lesson as it is from this dropdown you can remove sorts and change the order in which sorts are applied. I definitely feel a screenshot of the "Sort" menu would help here and/or the description could be expanded to more clearly describe what happens - but at a minimum we need to keep the original sentence

Reply:
When I say "change the order in which sorts are applied" this is not the same as the sort order for a particular column. If we have a project with an Author column and a Title column, and you have applied sorts on both columns, in the additional "Sort" drop down menu you can decide whether you do:

- Sort by title, then by author (so if two books had the same title but different authors, the two rows for that title would be next to each other in the sort, but ordered by the author name)
- Sort by author, then by title (so if one author had written multiple books, the rows for that author would be grouped together, ordered by the title of the book)